src/components/ui/ProgressBarStep.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import CircleIcon from '../ui/CircleIcon'
+import { theme } from './GlobalStyle'
+
+const PrimaryText = styled.div`
+  font-size: 15px;
+  color: ${({ color }) => color};
+`
+const StyledCircleIcon = styled(CircleIcon)`
+  border: ${({ theme }) => theme.secondaryText} 2px solid;
+`
+
+const ProgressBarStep = ({ label, stepNumber, className }) => (
+  <div className={className}>
+    <StyledCircleIcon backgroundColor={theme.secondaryBackground}>
+      <PrimaryText color={theme.secondaryText}>{stepNumber}</PrimaryText>
+    </StyledCircleIcon>
+    <PrimaryText color={theme.tertiaryText}>{label}</PrimaryText>
+  </div>
+)
+
+const StyledProgressBarStep = styled(ProgressBarStep)`
+  position: relative;
+  & + & {
+    flex-grow: 1;
+    display: flex;
+    align-items: flex-end;
+    flex-direction: column;
+  }
+
+  & + &::before {
+    top: calc(50% - 0.6rem);
+    right: 36px;
+    height: 100%;
+    position: absolute;
+    align-self: center;
+    content: '';
+    border-top: 2.5px solid black;
+    width: calc(100% - 34px);
+  }
+`
+StyledProgressBarStep.displayName = 'ProgressBarStep'
```

Comment:
Can you do the same rearrangement with the styled component as we did in ProgressBar?